src/main/java/AccountRetrievalTesting.java

```diff
@@ -0,0 +1,13 @@
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+public class AccountRetrievalTesting {
+
+    public static void main(String [] args) throws IOException, ExecutionException, InterruptedException {
```

Comment:
this is not a test; was it supposed to be?

Reply:
I was testing to see if I could retrieve information from the database. Its just that the amount of information retrieved might not always be the same since it depends on whether the database has been updated, so I can't really set an expected value.